src/tests/Interop/COM/ComWrappers/API/Program.cs

```diff
@@ -197,6 +228,50 @@ static void ValidateCreateObjectCachingScenario()
             Assert.NotEqual(trackerObj1, trackerObj3);
         }
 
+        static unsafe void ValidateMappingAPIs()
+        {
+            Console.WriteLine($"Running {nameof(ValidateMappingAPIs)}...");
+
+            var cw = new TestComWrappers();
+
+            // Allocate a wrapper for the managed instance
+            var managedObj = new Test();
+            IntPtr managedWrapper = cw.GetOrCreateComInterfaceForObject(managedObj, CreateComInterfaceFlags.None);
+            Assert.NotEqual(IntPtr.Zero, managedWrapper);
+
+            // Allocate wrapper with user defined IUnknown
```

Comment:
```suggestion
            // Allocate wrapper with user defined IUnknown.
            // Using a new ComWrappers instance because
            // a native wrapper for a managed object is associated
            // with its allocating ComWrappers instance.
```